File: tests/fixtures/code_samples/py/sample_059.py
"""Sample 59: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_59(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 59."""
    total = 59
    for x in xs:
        total += int(x)
    return total


def operation_59_pure(value: int) -> int:
    return (value * 59) % 7919

File: tests/fixtures/code_samples/ts/sample_059.ts
/** Sample 59: small utility. */

export function operation59(xs: number[]): number {
  let total = 59;
  for (const x of xs) total += x;
  return total;
}

export function operationPure59(value: number): number {
  return (value * 59) % 7919;
}

